daisy/instance.go

```diff
@@ -145,6 +145,12 @@ func (i *Instance) populateDisks(w *Workflow) dErr {
 			} else {
 				p.DiskType = fmt.Sprintf("projects/%s/zones/%s/diskTypes/%s", i.Project, i.Zone, p.DiskType)
 			}
+			parts := namedSubexp(diskTypeURLRgx, p.DiskType)
+			if parts["disktype"] == "local-ssd" {
+				d.AutoDelete = true
+				d.Type = "SCRATCH"
+				p.DiskName = ""
```

Comment:
You also need to allow for interface as there are two options (NVME and SCSI)

Reply:
If he doesn't define, [it's "SCSI" by default](https://developers.google.com/resources/api-libraries/documentation/compute/v1/python/latest/compute_v1.instances.html#insert).
But maybe you mean something else...

Reply:
I'm not checking that, these are just put here because they are required fields. Default interface type is scsi, so you don't have to set it. If you set interface to nvme that also works though